penguin/skills/discovery.py

```
@@ -2,13 +2,20 @@
 
 from __future__ import annotations
 
+import logging
 import os
+from importlib import resources as importlib_resources
 from pathlib import Path
 from typing import Any, Dict, Iterable, List, Optional, Tuple, Union
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)